docs/modules/cosmosdb.md

```diff
@@ -23,6 +23,10 @@ Provisions a new CosmosDB database account that can be accessed through any of t
 | `location` | `string` | The Azure region in which to provision applicable resources. | Required _unless_ an administrator has configured the broker itself with a default location. | The broker's default location, if configured. |
 | `resourceGroup` | `string` | The (new or existing) resource group with which to associate new resources. | N | If an administrator has configured the broker itself with a default resource group and nonde is specified, that default will be applied, otherwise, a new resource group will be created with a UUID as its name. |
 | `tags` | `map[string]string` | Tags to be applied to new resources, specified as key/value pairs. | N | Tags (even if none are specified) are automatically supplemented with `heritage: open-service-broker-azure`. |
+| `ipFilters` | `object` | IP Range Filter to be applied to new CosmosDB account | N | No Filters Applied |
+| `ipFilters.allowAccessFromAzure` | `string` | Specifies if Azure Services should be able to access the CosmosDB account.Valid valued are `""` (unspecified), `enabled`, or `disabled`. | N | If left unspecified, defaults to enabled. |
```

Comment:
> If left unspecified, defaults to enabled.

Is that contrary to what you told me current behavior is? (fwiw, I do think this is what we should default to.)

Reply:
Nope, so I probably need to update this comment.

Default behavior, if you don't provide *any* filters is that nothing is applied and it's open to the world.

*If* you provide any filters, it defaults to allow access from all of azure in addition to anything else you provide.

Reply:
imo, we should be more secure by default. So I like the doc as it is. Can we just bring the code in-line with the doc?